src/intentHandlers/ProjectDescription.js

```diff
@@ -0,0 +1,41 @@
+const SlackService = require('../services/SlackService.js');
+const JiraService = require('../services/JiraService.js');
+const DBService = require('../services/DBService.js');
+const async = require('asyncawait/async');
+const await = require('asyncawait/await');
+const Slackify = require('slackify-html');
+
+module.exports.process = async ((event, token) => {
+    // get the project key for the channel
+    const ContextResponse = await (DBService.retrieveChannelContext(event.channel)
+        .catch(error => {throw new Error(`Failed to fetch project key for channel context: ${error}`)}));
+
+    if (!ContextResponse){
+        throw new Error('Please set the channel project first');
```

Comment:
What does "Please set the channel project first" mean? Message is a bit confusing to me?

Reply:
Thanks, updated it to make it clearer